cgo.go

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+
+//go:build cgo || ios
+// +build cgo ios
```

Comment:
Just cgo

Reply:
Or `cgo && darwin`? If so, please add a suffix for this file

Reply:
It should just be cgo